articles/modules/ROOT/pages/realtime-querying-of-the-query-log-files.adoc

```diff
@@ -0,0 +1,85 @@
+= Realtime querying of the query.log files
+// CHANGE THIS, REMOVE COMMENTS BEFORE MERGE
+// ANY SLUG CHANGES WILL CAUSE POSTS TO BE RE-CREATED, BREAKING EXISTING LINKS
+:slug: realtime-querying-of-the-query-log-files
+:author: Dana Canzano
+:twitter: @your-twitter
```

Comment:
Please remove any comments, and remove any unneeded fields. The twitter line can probably be removed.